src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -116,7 +118,7 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      *
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, FileNotFoundException {
```

Comment:
Correct addition for the new exception.

Note that there are other methods which use the save() method......